Sources/PostgresNIO/New/Data/PostgresINET+PostgresCodable.swift

```diff
@@ -0,0 +1,66 @@
+public struct PostgresINET {
+    public let ipFamily: UInt8
+    public let netmaskLength: UInt8
+    public let isCIDR: Bool
+    public let addressLength: UInt8
+    public let ipAddress: [UInt8]
+}
```

Comment:
I think this is too close to the on the wire representation. In an ideal case this would be closer to the semantical meaning:

```swift
struct PostgresIPv4 {
  var value: (UInt8, UInt8, UInt8, UInt8)
}

struct PostgresIPv6 {
  var value: (UInt16, UInt16, UInt16, UInt16, UInt16, UInt16, UInt16, UInt16)
}

enum PostgresINET {
  case ipv4(PostgresIPv4, networkMask: UInt8?)
  case ipv4(PostgresIPv6, networkMask: UInt32?)
}
```

Since there are no currency types for this in Swift, we need to have our own minimal type for this.

Reply:
@fabianfett Without an explicit `isCIDR` property, would it be safe to infer that an instance uses CIDR notation based on whether the IPv4 network mask is 32 (or IPv6 network mask is 128)?